src/Uniswap.Data/Repositories/IExchangeRepository.cs

```diff
@@ -11,5 +11,6 @@ public interface IExchangeRepository
         Task<IEnumerable<IExchangeEntity>> GetAllAsync();
         Task<IExchangeEntity> GetLastCreatedAsync();
         Task Update(string exchangeId, decimal ethLiquidity, decimal tokenLiquidity, decimal totalSupply);
+        Task<IEnumerable<IExchangeEntity>> GetAsync(decimal minEthLiquidityAmount);
```

Comment:
I would propose to remove `GetAllAsync` method and make minEthLiquidityAmount parameter optional